src/editor/ui_menus/toolsize_menu.h

```diff
@@ -37,12 +36,10 @@ struct EditorToolsizeMenu : public UI::UniqueWindow {
 
 private:
 	EditorInteractive& eia() const;
-	void decrease_radius();
-	void increase_radius();
-
-	UI::Textarea textarea_;
-	UI::Button increase_, decrease_;
+	UI::SpinBox spinbox_;
 	uint32_t value_;
+	void changed_();
+	const uint32_t margin = 5;
```

Comment:
Just two codestyle nits:
```suggestion
	void changed();
	static constexpr uint32_t kMargin = 5;
```